.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - 1.13.x
+  - 1.14.x
```

Comment:
Are we officially on `go1.14` yet?

Reply:
All of our other services have moved CI go version to `go1.14`. copr is holding us back though for building RPMs with this version  

Unless it lets us install go from another registry, https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/g/ only has `go1.13.6` as latest.